dev/com.ibm.ws.logging.hpel_fat/fat/src/com/ibm/ws/logging/hpel/fat/HPELHideMessagesTest.java

```diff
@@ -79,8 +78,8 @@ public static void setUp() throws Exception {
         // Setting the server.xml with the hideMessages logging attribute
         server.updateServerConfiguration(new File(server.pathToAutoFVTTestFiles, "server-HPELHideMessagesTest.xml"));
         // Restart server
-        server.restartServer();
-
+        server.stopServer();
```

Comment:
Not sure why we need this change as `server.restartServer()` should do `server.stopServer()` and `server.startServer()`.